test/runtime/fake_vqeruntime.py

```diff
@@ -0,0 +1,99 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Fake runtime provider and VQE runtime."""
+
+from typing import Dict, Any
+import numpy as np
+from qiskit.circuit import QuantumCircuit
+from qiskit.opflow import PauliSumOp
+from qiskit.providers import Provider
+from qiskit_optimization.runtime import VQEProgramResult
+
+
+class FakeVQEJob:
```

Comment:
Would using the mock library be a better practice here? 
https://docs.python.org/3/library/unittest.mock.html

Reply:
Maybe, I'm not familiar with using the mock library yet and it seemed fairly straightforward to just implement these classes. Would it be easier or shorter with the mock library?